conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -11,4 +11,5 @@
 
 # Add all the application routes to the prod.routes file
 ->         /                          prod.Routes
-DELETE     /one-stop-shop-registration/test-only/delete-accounts        @controllers.test.TestOnlyController.deleteAccounts()
\ No newline at end of file
+DELETE     /one-stop-shop-registration/test-only/delete-accounts                          @controllers.test.TestOnlyController.deleteAccounts()
+GET        /one-stop-shop-registration/test-only/create-registrations/:startVrn/:endVrn/:commencementDate   @controllers.test.TestOnlyController.createRegistrations(startVrn:Int, endVrn: Int, commencementDate: LocalDate)
```

Comment:
I feel this should either be a PUT or POST

Reply:
done now